app/components/CompanyValues.tsx

```diff
@@ -0,0 +1,150 @@
+// LIBRARY IMPORTS
+import {
+  Box,
+  Flex,
+  Heading,
+  HStack,
+  Icon,
+  Spacer,
+  Text,
+  VStack,
+} from "@chakra-ui/react";
+import Image from "next/image";
+import { FaRegClock, FaPeopleGroup, FaRegStar } from "react-icons/fa6";
+
+// LOCAL IMPORTS
+import gearBackground from "../../public/images/machine-gear-background.png";
+
+// VALUES COMPONENT DEFINITION
+export default function CompanyValues() {
+  return (
+    <>
+      <Image
+        src={gearBackground}
+        placeholder="blur"
+        fill
+        sizes="100vw"
+        style={{
+          objectFit: "cover",
+          zIndex: -1,
+        }}
+      />
+      <Box
+        bgImage={gearBackground}
+        backgroundRepeat="no-repeat"
+        backgroundSize="cover"
+        p={10}
+        bg="brand.backgroundDark"
+        opacity="90%"
+      >
+        <VStack spacing={8}>
+          <Heading color="white" textAlign="center" mb={4}>
```

Comment:
I would increase vertical spacing around the heading maybe by adding top and bottom padding to get it closer to Figma design